Xamarin.Forms.Core/Frame.cs

```diff
@@ -87,5 +87,11 @@ void IBorderElement.OnBorderColorPropertyChanged(Color oldValue, Color newValue)
 		bool IBorderElement.IsBorderColorSet() => IsSet(BorderColorProperty);
 
 		bool IBorderElement.IsBorderWidthSet() => false;
+
+		static void OnCornerRadiusChanged(BindableObject bindable, object oldValue, object newValue)
+		{
+			if (bindable is Frame frame)
+				frame.IsClippedToBounds |= (float)newValue > 0f;
```

Comment:
only set IsClippedToBounds to 'true' if it never was set by the user. do not reset it to false, as we do not know who set it to true

Reply:
Or even better, avoid this side effect, and handle this in the android renderer

Reply:
@StephaneDelcroix  Any tip to how detect user change?  I mean, i thinking to track `OnPropertyChanged` and keep this information in a variable (like `bool clippedToBoundsUserChanged`)

Reply:
@StephaneDelcroix I found this solution: (applied at #8032 )
https://github.com/xamarin/Xamarin.Forms/pull/8032/files#diff-7e99600fa8a972c09f94b521218f1e0eR241